Script error in console when `highlighting code with language` in CodePair

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->


**What happened**:

In specific scenarios, a script error occurs in the console when attempting to highlight code with a designated language in CodePair. The error message in the console reads as follows:

```
console.js:36 Error highlighting code with language '': Error: Unknown language: `` is not registered
    at iOe.Yft [as highlight] (core.js:99:13)
    at Object.highlight (Preview.tsx:32:15)
    at c0.fence (renderer.mjs:43:27)
    at renderer.js:111:47
    at Array.forEach (<anonymous>)
    at renderer.js:103:16
    at incremental-dom-cjs.js:879:5
    at t (incremental-dom-cjs.js:851:26)
    at Preview.tsx:97:3
    at N6 (react-dom.production.min.js:243:332)
    at yO (react-dom.production.min.js:285:111)
    at bJ (react-dom.production.min.js:272:185)
    at j1 (react-dom.production.min.js:127:105)
    at xCe (react-dom.production.min.js:282:470)
    at Lb (react-dom.production.min.js:280:398)
    at fde (react-dom.production.min.js:269:269)
    at F (scheduler.production.min.js:13:203)
    at MessagePort.ie (scheduler.production.min.js:14:128)
```

**What you expected to happen**:

I expected that the code highlighting feature would work without any errors in the console, ensuring a smooth experience for developers when using the CodePair tool.

**How to reproduce it (as minimally and precisely as possible)**:

1. Use the CodePair feature to highlight code with a specified language. or visit [this page](https://codepair.yorkie.dev/yorkie@ossca-2025/68af134c84afa954c7459993)
2. Observe the console for any script errors that appear during this process.

**Anything else we need to know?**:

**Environment**:
- Operating system: Mac
- Browser and version: Chrome latest
- CodePair version: v0.1.42

--- a/frontend/src/components/editor/Preview.tsx
+++ b/frontend/src/components/editor/Preview.tsx
@@ -61,6 +61,9 @@ const md = new MarkdownIt({
 	linkify: true,
 	breaks: true,
 	highlight: (code: string, lang: string): string => {
+		if (lang.trim() === "") {
+			return `<pre><code>${md.utils.escapeHtml(code)}</code></pre>`;
+		}
 		try {
 			return `<pre class="language-${lang}"><code>${toHtml(
 				// eslint-disable-next-line @typescript-eslint/no-explicit-any
@@ -68,7 +71,7 @@ const md = new MarkdownIt({
 			)}</code></pre>`;
 		} catch (error) {
 			console.error(`Error highlighting code with language '${lang}':`, error);
-			return `<pre class="language-"><code>${md.utils.escapeHtml(code)}</code></pre>`;
+			return `<pre><code>${md.utils.escapeHtml(code)}</code></pre>`;
 		}
 	},
 })

